projects/frontend/data-pipelines/gui/e2e/integration/app/getting-started/getting-started.spec.js

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2021 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/// <reference types="cypress" />
+import { DataPipelinesBasePO } from '../../../support/application/data-pipelines-base.po';
```

Comment:
BTW 

the PageObject patterns(https://martinfowler.com/bliki/PageObject.html) is used in the Cypress tests.

May be it worth mentioning this in the VEP in order to keep future tests development consistent. .

Reply:
Added to address in https://github.com/vmware/versatile-data-kit/issues/1643